feature/feature.go

```diff
@@ -20,12 +20,16 @@ import (
 )
 
 const (
-// Every capg-specific feature gate should add method here following this template:
-//
-// // owner: @username
-// // alpha: v1.X
-// MyFeature featuregate.Feature = "MyFeature".
+	// Every capg-specific feature gate should add method here following this template:
```

Comment:
We can remove this as it will be added in PR5

Reply:
I'm also wondering do we need to worry about the CAPI machinepool feature flag?

Reply:
But we can worry about that in the next PR :)

Reply:
change reverted :)